src/lazy/text/raw/v1_1/reader.rs

```diff
@@ -34,6 +34,84 @@ pub struct LazyRawTextReader_1_1<'data> {
     local_offset: usize,
 }
 
+impl<'data> LazyRawReader<'data, TextEncoding_1_1> for LazyRawTextReader_1_1<'data> {
```

Comment:
🗺️ I moved this `impl` block so it would be right below the definition of `LazyRawTextReader_1_1`. There are no logic changes.